sbnanaobj/StandardRecord/SRBlip.h

```diff
@@ -0,0 +1,44 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRBlip.h
+// \brief   SRBlip object for localized energy deposits in bulk LAr
+// \author  $Author: jmclaughlin2@illinoistech.edu
+////////////////////////////////////////////////////////////////////////
```

Comment:
Same Doxygen format:
```suggestion
/**
 * @file    SRBlip.h
 * @brief   SRBlip object for localized energy deposits in bulk LAr
 * @author  jmclaughlin2@illinoistech.edu
 */
```